aiohttp/web_urldispatcher.py

```diff
@@ -690,10 +690,7 @@ def _resolve_path_to_response(self, unresolved_path: Path) -> StreamResponse:
         except PermissionError as error:
             raise HTTPForbidden() from error
 
-        # Not a regular file or does not exist.
-        if not file_path.is_file():
-            raise HTTPNotFound()
```

Comment:
This breaks some code in aiohttp-devtools:
https://github.com/aio-libs/aiohttp-devtools/blob/270aa58741b9aad268e32cb7bd755fda4c562731/aiohttp_devtools/runserver/serve.py#L404

Any suggestions on how to proceed?